File: algorithms/warmup/simple-array-sum/simple-array-sum.php
<?php
$_fp = fopen("php://stdin", "r");
/* Enter your code here. Read input from STDIN. Print output to STDOUT */
$size = (int)fgets($_fp);
$numbers = fgets($_fp);
$array = split(" ", $numbers, $size);
$sum = 0;
for($i=0;$i<count($array);$i++){
    $sum += (int)$array[$i];
}
print $sum;
?>

File: algorithms/warmup/simple-array-sum/simple-array-sum.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {
    int n;
    scanf("%d", &n);
    int my_array[n];
    int i, sum = 0;
    for (i=0;i<n;i++){
        scanf("%d", &my_array[i]);
        sum += my_array[i];
    }
    printf("%d", sum);
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    return 0;
}
